claudio-plugin/skills/package-request/scripts/_validate.sh

```
@@ -0,0 +1,155 @@
+        existing_tickets=$(echo "$response" | jq --arg url "$jira_url" '[.issues[] | {
+            ticket_id: .key,
+            ticket_url: ($url + "/browse/" + .key),
+            summary: (.fields.summary // "")
+        }]')
+        echo "DUPLICATE"
+        jq -n --arg pkg "$summary" \
+              --argjson tickets "$existing_tickets" \
+              --argjson total "$total" \
+              --arg days "$duplicate_days" \
+            '{
+                message: ("A request overlapping with '" + $pkg + "' was submitted in the past " + $days + " days."),
```

Shellcheck (0.11.0)

[warning] pkg is referenced but not assigned.

(SC2154)